app/Commands/StoreFootballTeams.php

```
@@ -0,0 +1,124 @@
+    {
+        $this->service = new FootballService();
+        $this->leagueModel = new \App\Models\FootballLeagues();
+        $this->teamModel = new \App\Models\FootballTeams();
+    }
+
+    /**
+     * Actually execute a command.
+     *
+     * @param array $params
+     */
+    public function run(array $params)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$params'. (Unused Code Rules)

(UnusedFormalParameter)